### PR TITLE
chore: add changelog for 1.36.0

### PR DESCRIPTION
**Pull request by code-asher**
I excluded commits that have been cherry picked and released into the 1.35 line but there are some that have been cherried but not released yet.  I left them in here but am not sure if that is the right move.  Feel free to push up anything I missed or reword things, or leave comments and I will make the changes.

- [x] Update description with date

### CODE REVIEW

File: changelog/1.36.0.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+### Breaking changes ❗
+
+There are no breaking changes in 1.36.0
+
+### Features ✨
+
+- infra: Add sessionAffinity to coderd service.
+- infra: Allow older verions of curl for self-contained builds.
+- infra: Include coder-cli asset to coderd-run-dev for self-contained builds.
+- infra: Allow workspace builds even if a webhook fails.
```
Comment (spikecurtis) [inline changelog/1.36.0.md:15]:
```suggestion
- infra: Allow workspace template builds even if user does not have permission to add webhooks on their git repo.
```

File: changelog/1.36.0.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.36.0
+
+### Features ✨
+
+- infra: Add sessionAffinity to coderd service.
+- infra: Allow older verions of curl for self-contained builds.
+- infra: Include coder-cli asset to coderd-run-dev for self-contained builds.
```
Comment (spikecurtis) [inline changelog/1.36.0.md:14]: Development only.  I don't think this should be included in customer-facing release notes.

File: changelog/1.36.0.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- web: Fixed relative links on embedded docs.
+- infra: Added a new background job manager.
+- web: Allow org members to view activity metrics.
+- infra: Prevent a panic when updating a template.
+- infra: Return the updated user after an update instead of the cached user.
+- infra: Fix SSH connections not being audit logged.
+
+### Security updates 🔐
+
+- infra: Make Coder services inside workspaces listen on 127.0.0.1.
```
Comment (spikecurtis) [inline changelog/1.36.0.md:31]:
```suggestion
- infra: Fixed an issued where Coder services inside the workspace could be reached via the network from outside in some environments
```

File: changelog/1.36.0.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+There are no breaking changes in 1.36.0
+
+### Features ✨
+
+- infra: Add sessionAffinity to coderd service.
+- infra: Allow older verions of curl for self-contained builds.
+- infra: Include coder-cli asset to coderd-run-dev for self-contained builds.
+- infra: Allow workspace builds even if a webhook fails.
+
```
Comment (spikecurtis) [inline changelog/1.36.0.md:16]:
```suggestion
- infra: API request latency Prometheus metrics are broken out by route 
```

Comment (johnstcn) [inline changelog/1.36.0.md:16]:
```suggestion
- infra: Adds Prometheus metrics around background job execution.
```

Comment (johnstcn) [inline changelog/1.36.0.md:16]:
```suggestion
- infra: Adds support for cached CVMs with 5.15 and 5.16 kernel versions.
```

File: changelog/1.36.0.md (hunk 2 above)
Comment (code-asher) [reply changelog/1.36.0.md:14]: Oof, I misunderstood the change.  Will remove!

File: changelog/1.36.0.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  on a workspace that has shut off.
+- infra: Fixed a goroutine leak.
+- web: Fixed relative links on embedded docs.
+- infra: Large numbers of concurrent workspace builds are distributed more
+  evenly amongst multiple coderd replicas.
+- web: Allow org members to view activity metrics.
+- infra: Prevent a panic when updating a template.
+- infra: Return the updated user after an update instead of the cached user.
+- infra: Fix SSH connections not being audit logged.
+
```
Comment (spikecurtis) [inline changelog/1.36.0.md:32]:
```suggestion
- infra: Fixed an issue where temporary pods created during build did not have templates applied
```

File: changelog/1.36.0.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+- web: Fixed relative links on embedded docs.
+- infra: Large numbers of concurrent workspace builds are distributed more
+  evenly amongst multiple coderd replicas.
+- web: Allow org members to view activity metrics.
+- infra: Prevent a panic when updating a template.
+- infra: Return the updated user after an update instead of the cached user.
+- infra: Fix SSH connections not being audit logged.
+- infra: Fixed an issue where temporary pods created during build did not have
+  templates applied.
+
```
Comment (johnstcn) [inline changelog/1.36.0.md:34]:
```suggestion
- infra: Fixed an issue where the hostname was set to `workspace` for CVM workspaces
  irrespective of the workspace name. 

```

File: changelog/1.36.0.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
+---
+title: "1.36.0"
+description: "Released on TBD"
```
Comment (johnstcn) [inline changelog/1.36.0.md:3]:
```suggestion
description: "Released on 10/26/2022"
```

File: changelog/1.36.0.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+- infra: Fixed an issued where Coder services inside the workspace could be
+  reached via the network from outside in some environments.
+
+### Known Issues ℹ️
+
+- On AWS EKS, cached CVMs are known to not work with more recent versions of the
+  `Ubuntu2004` AMI family that include kernel version `5.15-aws`. Workaround:
+  roll back to a version that includes an earlier kernel version (for example,
+  `ubuntu-eks/k8s_1.22/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-20220706`)
+  or build a custom AMI with the generic kernel.
```
Comment (johnstcn) [inline changelog/1.36.0.md:48]: ☝️

Comment (spikecurtis) [reply changelog/1.36.0.md:48]: Out of curiosity, does 22.04 work fine?

Comment (johnstcn) [reply changelog/1.36.0.md:48]: > Out of curiosity, does 22.04 work fine?

I'm not aware of any 22.04 AMIs published by Amazon.

File: changelog/1.36.0.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+### Bug fixes 🐛
+
+- web: Fixed the "back to site" button that appears when trying to access an IDE
+  on a workspace that has shut off.
+- infra: Fixed a goroutine leak.
+- web: Fixed relative links on embedded docs.
+- infra: Large numbers of concurrent workspace builds are distributed more
+  evenly amongst multiple coderd replicas.
+- web: Allow org members to view activity metrics.
+- infra: Prevent a panic when updating a template.
```
Comment (Emyrk) [inline changelog/1.36.0.md:29]: This panic was never released
```suggestion
```

File: changelog/1.36.0.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+- infra: Return the updated user after an update instead of the cached user.
```
Comment (Emyrk) [inline changelog/1.36.0.md:30]: Idk if anyone relied on this? We can probably not include this. Might look better to have less "bugs". It was causing a unit test failure.
```suggestion
```

**Review by Emyrk — APPROVED** [review]

**Pull request merged**